.github/workflows/build-os-image.yml

```
@@ -28,13 +38,13 @@ jobs:
           ref: ${{ github.head_ref }}
 
       - name: Build bootstrapper
-        if: ${{ inputs.debug == false }}
+        if: ${{ inputs.stream != 'debug' }}
```

```suggestion
        if: inputs.stream != 'debug'
```
No need for `${{` in if